javagen/src/main/java/com/azure/autorest/model/javamodel/JavaPackage.java

```diff
@@ -178,6 +179,21 @@ protected void addJavaFile(JavaFile javaFile) {
         javaFiles.add(javaFile);
     }
 
+    public void addProtocolExamples(ClientMethod method, MethodGroupClient client, String builderName) {
+        method.getProxyMethod().getExamples().forEach((name, example) -> {
+            String filename = CodeNamer.getTypeName(name);
```

Comment:
Maybe just `toPascalCase(removeInvalidCharacters`. Avoid include the preprocess package.

Reply:
Updated